modules/seqexec/web/server/src/main/resources/app.conf

```diff
@@ -67,6 +67,7 @@ seqexec-engine {
     smartGCalHost = "gsodbtest.gemini.edu"
     # Tmp file for development
     smartGCalDir = "/tmp/smartgcal"
-    gpiUrl = "vm://gpi?marshal=false&broker.persistent=false"
-    gpiGDS = "http://localhost:8888/xmlrpc"
+    #gpiUrl = "vm://gpi?marshal=false&broker.persistent=false"
+    gpiUrl = "failover:(tcp://172.17.107.50:61616)?timeout=4000"
+    gpiGDS = "http://172.17.107.50:8888/xmlrpc"
```

Comment:
These changes are from the testing with the real instruments. They must be reverted before the commit.